src/ActiveRecord.php

```diff
@@ -46,7 +47,7 @@ class ActiveRecord extends BaseActiveRecord
      */
     public static function getDb()
     {
-        return Yii::$app->get('redis');
+        return Instance::ensure('redis', Connection::className());
```

Comment:
This may break BC and it is inconsistent with `yii\db\ActiverRecord` which always point to `Yii::$app->db` (https://github.com/yiisoft/yii2/blob/b79fe7d106b8f901d3f3f2fd01eae811ce2c34ec/framework/db/ActiveRecord.php#L135). You should override this method in your model if you want to use different approach.